.circleci/compute_merge_commit.sh

```diff
@@ -40,6 +40,11 @@ MERGE_BRANCH="refs/pull/$PR_NUMBER/merge"
 echo $(GREEN "Computing merge SHA of $MERGE_BRANCH...")
 CIRCLE_MERGE_SHA="$(git ls-remote https://github.com/ampproject/amphtml.git "$MERGE_BRANCH" | awk '{print $1}')"
 
+# TODO(danielrozenberg): after #33708 is submitted, remove local directory copy and only store one in the workspace.
```

Comment:
Reminder to update the logging statement on line 49 when this happens.

Reply:
Updated it now, might as well... It's technically correct :)